db/models.py

```diff
@@ -1,4 +1,10 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` model extends `AbstractUser` but does not add any additional fields or methods. Ensure that this is intentional and that the `AUTH_USER_MODEL` setting in your Django settings is set to this `User` model if you intend to use it as the custom user model.